zcu/encryption.py

```diff
@@ -3,11 +3,19 @@
 import struct
 from io import BytesIO
 from Cryptodome.Cipher import AES
+from hashlib import sha256
 
 from . import constants
 
 
-def aes_decrypt(cipher, aes_key):
+def digi_key(serial):
+    plain_key = constants.DIGIMOBIL_BASE_KEY + serial
+    plain_iv = constants.DIGIMOBIL_BASE_IV + serial
+    key = sha256(plain_key.encode('utf-8')).digest()
+    iv = sha256(plain_iv.encode('utf-8')).digest()
+    return (key,iv)
```

Comment:
super nitpick, but can you add a space between `(key, iv)` just stuff that'll come up in the linter